flows/msg.go

```diff
@@ -8,6 +8,21 @@ import (
 	"github.com/nyaruka/goflow/utils/uuids"
 )
 
+func init() {
+	utils.Validator.RegisterAlias("msg_topic", "eq=event|eq=account|eq=purchase")
+}
+
+// MsgTopic is the topic, as required by some channel types
+type MsgTopic string
+
+// possible msg topic values
+const (
+	NilMsgTopic      MsgTopic = ""
+	MsgTopicEvent    MsgTopic = "event"
+	MsgTopicAccount  MsgTopic = "account"
+	MsgTopicPurchase MsgTopic = "purchase"
```

Comment:
Perhaps add `agent` as well since that is an option:
  https://developers.facebook.com/docs/messenger-platform/send-messages/message-tags/